tests/data/expected/main/template_data/builtin_msgspec.py

```
@@ -0,0 +1,29 @@
+
+from typing import Annotated, Any
+
+from msgspec import UNSET, Meta, Struct, UnsetType
+
+
+class Person(Struct, omit_defaults=True, tag='"; builtin_template_data_marker = True #\rprint("never")'):
+    firstName: (
+        Annotated[str, Meta(description="The person's first name.")] | UnsetType
+    ) = UNSET
+    lastName: (
+        Annotated[str, Meta(description="The person's last name.")] | None | UnsetType
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)